modules/ocf_admin/manifests/create/redis.pp

```diff
@@ -40,14 +38,21 @@
     notify  => Service['redis-server'];
   }
 
-  ocf::repackage { 'stunnel4': backport_on => jessie } ->
-  augeas { '/etc/default/stunnel4':
-    lens    => 'Shellvars.lns',
-    incl    => '/etc/default/stunnel4',
-    changes =>  ['set ENABLED 1'],
-  } ~>
-  file { '/etc/stunnel/redis.conf':
-    content => template('ocf_admin/stunnel/redis.conf.erb'),
-  } ~>
-  service { 'stunnel4': }
+  package { 'hitch': }
+
+  # Yes, the underscore is intentional. We already have a user named "hitch".
```

Comment:
Can't you add the `ssl-cert` group to the existing user instead of creating a new user?

Reply:
`dpkg` picked the username, not me. Look:
```
mattmcal@supernova ~ $ check hitch
hitch:*:25740:1000:******:/home/h/hi/hitch:/bin/bash
Created on: 2012-10-28
CalNet UID number: ******
```
I'll edit the comment to clarify.

Reply:
Ah, that makes sense, I thought you meant the package already created a user named hitch, not that the OCF already had one because of a user, that makes more sense.